client/components/googleMaps.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Loader } from '@googlemaps/js-api-loader';
+
+const loader = new Loader({
+  apiKey: process.env.GOOGLE_API,
+  version: 'weekly',
+  libraries: ['places', 'marker', 'drawing']
+});
+
+export default class GoogleMaps extends React.Component {
+  constructor(props) {
+    super(props);
+    this.googleMapDiv = React.createRef();
+    this.createMap = this.createMap.bind(this);
+  }
+
+  componentDidMount() {
+    this.createMap();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.results !== prevProps.results) {
+      this.createMap();
+    }
+  }
+
+  createMap() {
+    const states = this.props.results;
+    const mapOptions = {
+      center: { lat: 37.0902, lng: -95.7129 },
+      zoom: 4,
+      streetViewControl: false,
+      mapTypeControl: false
+    };
+    loader.load().then(google => {
+      const map = this.map = new window.google.maps.Map(this.googleMapDiv.current, mapOptions);
+      this.setState({
+        google,
+        map
+      });
+      return map;
+    })
+      .then(map => {
+        if (states.length !== 0) {
+          const bound = new window.google.maps.LatLngBounds();
+
+          states.map(state => {
```

Comment:
Since we are not using the new array created by `.map()`, it's best to use `.forEach()` instead.

From MDN:

<img width="776" alt="image" src="https://user-images.githubusercontent.com/7432943/200965240-516a6919-bfb8-434f-a8f9-f802a121173c.png">